tests/particle_output_multiple_formats/statistics

```diff
@@ -0,0 +1,16 @@
+# 1: Time step number
+# 2: Time (seconds)
+# 3: Number of mesh cells
+# 4: Number of Stokes degrees of freedom
+# 5: Number of temperature degrees of freedom
+# 6: Number of degrees of freedom for all compositions
+# 7: Iterations for temperature solver
+# 8: Iterations for composition solver 1
+# 9: Iterations for Stokes solver
+# 10: Velocity iterations in Stokes preconditioner
+# 11: Schur complement iterations in Stokes preconditioner
+# 12: Time step size (seconds)
+# 13: Number of advected particles
+# 14: Particle file name(s)
```

Comment:
Thinking more about it, I do not think it is a good idea to assume the output directory stays constant (one can change that over restarts, and that would lead to two different columns). So it needs to appear as a value in each line. And if it needs to appear in every line, it might as well be in a combined column with the file name. Just my opinion though.